Since we are using skaknew's uskak.fd we need to adapt skak.sty as described
in the skaknew readme.
---
 skak.sty |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: texlive-extra-2007.dfsg.1/texmf-dist/tex/latex/skak/skak.sty
===================================================================
--- texlive-extra-2007.dfsg.1.orig/texmf-dist/tex/latex/skak/skak.sty	2007-07-04 09:01:21.000000000 +0200
+++ texlive-extra-2007.dfsg.1/texmf-dist/tex/latex/skak/skak.sty	2007-11-13 14:42:38.000000000 +0100
@@ -1456,7 +1456,7 @@
 \newlength{\showlength}
 \newlength{\ranklift}
 
-\def\setup@showboard#1{\font\Skak=skak#1%
+\def\setup@showboard#1{\font\Skak=SkakNew-Diagram at#1pt%
   \setlength{\squarelength}{#1pt}%
   \square@multiplier=#1
   % ps setup
